backoffice/tests/UnitTests/Services/DroitServiceUnitTests.cs

```diff
@@ -164,5 +164,27 @@ public async Task ExportDroitsAsync_ListOfDroits_ReturnsData()
             // Assert
             Assert.NotEmpty(data);
         }
+        
+        [Fact]
+        public async Task ExportDroitsAsync_ListOfDroits_ReturnsCorrectData()
+        {
+            // Given
+            var droitSearchForm = new DroitSearchForm();
+            var droitsQueryable = new List<Droit>()
+            {
+                new() {Id = Guid.NewGuid(), Reference = "Ref1"},
+                new() {Id = Guid.NewGuid(), Reference = "Ref2"},
+            }.AsQueryable();
+
+            _mockRepo.Setup(r => r.GetDroitsWithAssociations()).Returns(droitsQueryable);
+            
+            // When
+            var data = await _service.ExportAsync(droitSearchForm);
+            
+            // Assert
+            Assert.Contains("Ref1", System.Text.Encoding.UTF8.GetString(data));
```

Comment:
Probably worth pulling out the text once to compare against 

 var data = await _service.ExportAsync(droitSearchForm);
 var dataString = System.Text.Encoding.UTF8.GetString(data)
 
 
 //Assert
 
 Assert.Contains("x", dataString);
 
 Assert.Contains("y", dataString);